src/mudlet-lua/lua/Other.lua

```diff
@@ -418,6 +418,10 @@ function speedwalk(dirString, backwards, delay, show)
   local walkdelay = delay
   if show ~= false then show = true end
   local walklist = {}
+  local long_dir = {north = 'n', south = 's', east = 'e', west = 'w', up = 'u', down = 'd'}
```

Comment:
Replacing all occurrences of `down` could be troublesome. How do you know I want to go `down` and not `down, out, west, north`?

Reply:
Coincidentally, the script as it was before this change would have also had trouble with that usage. If you note the patterns used to pull apart the directions, o isn’t a valid second character, so it would grab the d as a move, then it would grab ow (because w is a valid second character), and then it would get n.

So while we can imagine a hypothetical function that would have parsed “down” to give “down, out, west, north”, and therefore would have that functionality broken by this replacement, this is not such a function. If you want to perform those four moves here, you need to break it up somehow, which also fits with how it is described in the wiki.

Reply:
True... Since we don't break it any more than it was before, I'm okay with this change. But ultimately this should probably be fixed at some point. We should probably add an issue to remember this.